iOSIDPTemplate/template.js

```diff
@@ -38,7 +38,7 @@ function prepare(config, replaceInFiles, moveFile, removeFile) {
     // Values in template
     var templateAppName =  'Authenticator';
     var templatePackageName = 'com.salesforce.Authenticator';
-    var templateOrganization = 'Authenticator';
```

Comment:
This is the string we are searching in the xcode project file and replacing with what ever value the user provides when running forceios.
If you change it here, the generated template won't get the value provided by the user.

Reply:
The problem was that the script was replacing it as a partial string, so it corrupted the value that I provided when it named the project folder.

Reply:
+1. I don't think we want to change this.

Reply:
If we don't change it (or the way it's used), I believe that forceios createwithtemplate will fail for any app that uses "Authenticator" in its title.

Reply:
It's a good idea to have a very unique organization name.
However, the value template.js looks for, needs to actually be in project.pbxproj
Currently the value is wrong: https://github.com/forcedotcom/SalesforceMobileSDK-AuthTemplates/blob/master/iOSIDPTemplate/Authenticator.xcodeproj/project.pbxproj#L187

For comparison, in the default iOS native template, the org name is iOSNativeTemplateOrganizationName.
And it's actually in project.pbxproj: https://github.com/forcedotcom/SalesforceMobileSDK-Templates/blob/master/iOSNativeTemplate/iOSNativeTemplate.xcodeproj/project.pbxproj#L150